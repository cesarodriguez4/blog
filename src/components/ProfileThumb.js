import React, { Component } from 'react';
import { Button, Typography } from 'material-ui';

const imgUrl = `https://pbs.twimg.com/profile_images/818531326970265601/AVd-m6Cd_400x400.jpg`;
const imgStyle = {
  width: '100%',
  borderRadius: '50%'
};
export default class ProfileThumb extends Component {
  render() {
  	return (
      <div>
      	<img alt="Cesar Jr Rodriguez" src={imgUrl} style={imgStyle}></img>
      	<Typography className='mt-2 mb-2' align='center' type='title'><strong>Cesar Jr Rodriguez</strong></Typography>
      	<Typography align='center' type='caption'>Desarrollador de Software</Typography>
        <Typography align='center' type='caption'>Estudiante de Ingenier√≠a Civil en <strong>Universidad de Carabobo</strong></Typography>
      	  <Button><i className="fa fa-google fa-2x"></i></Button>
      	  <Button><i className="fa fa-facebook fa-2x"></i></Button>
      	  <Button><i className="fa fa-twitter fa-2x"></i></Button>
          <Button><i className="fa fa-stack-overflow fa-2x"></i></Button>
          <Button><i className="fa fa-linkedin fa-2x"></i></Button>
          <Button><i className="fa fa-github fa-2x"></i></Button>
        <Typography type='subheading' align='center'>Actualmente:</Typography>
        <Typography align='center'>Programador Web
          en <strong>ISMCenter</strong>
          </Typography>
        <Button className='container mt-2 bg-primary text-white' raised>Contratarme</Button>
      </div>

  		);
  }
}