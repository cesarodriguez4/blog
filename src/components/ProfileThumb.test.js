import React from 'react';
import ProfileThumb from './ProfileThumb';
import renderer from 'react-test-renderer';
import { MuiThemeProvider } from 'material-ui/styles';

it('render the profile area successfully', () => {
  const profile = renderer.create(
  	<MuiThemeProvider><ProfileThumb></ProfileThumb></MuiThemeProvider>).toJSON();
  expect(profile).toMatchSnapshot();
});