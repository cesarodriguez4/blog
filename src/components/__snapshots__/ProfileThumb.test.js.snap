// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render the profile area successfully 1`] = `
<div>
  <img
    alt="Cesar Jr Rodriguez"
    src="https://pbs.twimg.com/profile_images/818531326970265601/AVd-m6Cd_400x400.jpg"
    style={
      Object {
        "borderRadius": "50%",
        "width": "100%",
      }
    }
  />
  <h2
    className="MuiTypography-root-233331062 MuiTypography-title-2106356098 MuiTypography-alignCenter-345123901 mt-2 mb-2"
  >
    <strong>
      Cesar Jr Rodriguez
    </strong>
  </h2>
  <span
    className="MuiTypography-root-233331062 MuiTypography-caption-2998621467 MuiTypography-alignCenter-345123901"
  >
    Desarrollador de Software
  </span>
  <span
    className="MuiTypography-root-233331062 MuiTypography-caption-2998621467 MuiTypography-alignCenter-345123901"
  >
    Estudiante de Ingenier√≠a Civil en 
    <strong>
      Universidad de Carabobo
    </strong>
  </span>
  <button
    className="MuiButtonBase-root-3177716317 MuiButton-root-325725483"
    disabled={false}
    onBlur={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchStart={[Function]}
    role="button"
    tabIndex="0"
    type="button"
  >
    <span
      className="MuiButton-label-49836587"
    >
      <i
        className="fa fa-google fa-2x"
      />
    </span>
    <span
      className="MuiTouchRipple-root-3868442396"
    />
  </button>
  <button
    className="MuiButtonBase-root-3177716317 MuiButton-root-325725483"
    disabled={false}
    onBlur={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchStart={[Function]}
    role="button"
    tabIndex="0"
    type="button"
  >
    <span
      className="MuiButton-label-49836587"
    >
      <i
        className="fa fa-facebook fa-2x"
      />
    </span>
    <span
      className="MuiTouchRipple-root-3868442396"
    />
  </button>
  <button
    className="MuiButtonBase-root-3177716317 MuiButton-root-325725483"
    disabled={false}
    onBlur={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchStart={[Function]}
    role="button"
    tabIndex="0"
    type="button"
  >
    <span
      className="MuiButton-label-49836587"
    >
      <i
        className="fa fa-twitter fa-2x"
      />
    </span>
    <span
      className="MuiTouchRipple-root-3868442396"
    />
  </button>
  <button
    className="MuiButtonBase-root-3177716317 MuiButton-root-325725483"
    disabled={false}
    onBlur={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchStart={[Function]}
    role="button"
    tabIndex="0"
    type="button"
  >
    <span
      className="MuiButton-label-49836587"
    >
      <i
        className="fa fa-stack-overflow fa-2x"
      />
    </span>
    <span
      className="MuiTouchRipple-root-3868442396"
    />
  </button>
  <button
    className="MuiButtonBase-root-3177716317 MuiButton-root-325725483"
    disabled={false}
    onBlur={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchStart={[Function]}
    role="button"
    tabIndex="0"
    type="button"
  >
    <span
      className="MuiButton-label-49836587"
    >
      <i
        className="fa fa-linkedin fa-2x"
      />
    </span>
    <span
      className="MuiTouchRipple-root-3868442396"
    />
  </button>
  <button
    className="MuiButtonBase-root-3177716317 MuiButton-root-325725483"
    disabled={false}
    onBlur={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchStart={[Function]}
    role="button"
    tabIndex="0"
    type="button"
  >
    <span
      className="MuiButton-label-49836587"
    >
      <i
        className="fa fa-github fa-2x"
      />
    </span>
    <span
      className="MuiTouchRipple-root-3868442396"
    />
  </button>
  <h3
    className="MuiTypography-root-233331062 MuiTypography-subheading-2119063702 MuiTypography-alignCenter-345123901"
  >
    Actualmente:
  </h3>
  <p
    className="MuiTypography-root-233331062 MuiTypography-body1-2814455093 MuiTypography-alignCenter-345123901"
  >
    Programador Web en 
    <strong>
      ISMCenter
    </strong>
  </p>
  <button
    className="MuiButtonBase-root-3177716317 MuiButton-root-325725483 MuiButton-raised-1110834863 container mt-2 bg-primary text-white"
    disabled={false}
    onBlur={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchStart={[Function]}
    role="button"
    tabIndex="0"
    type="button"
  >
    <span
      className="MuiButton-label-49836587"
    >
      Contratarme
    </span>
    <span
      className="MuiTouchRipple-root-3868442396"
    />
  </button>
</div>
`;
